File: WebAPI/Models/UserModel.cs
using System;

namespace WebAPI.Models
{
    public class UserModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Address { get; set; }
        public DateTime Created { get; set; }
        public string Phone { get; set; }
    }
}

File: WebAPI/ClientApp/src/app/models/UserModel.ts
export class UserModel {
    public id: number;
    public name: string;
    public email: string;
    public address: string;
    public created: Date;
    public phone: string;

    public constructor(_id: number, _name: string, _email: string, _address: string, _created: Date, _phone: string) {
        this.id = _id;
        this.name = _name;
        this.email = _email;
        this.address = _address;
        this.created = _created;
        this.phone = _phone;
    }
}